src/parsers/manifest/smooth/create_parser.ts

```diff
@@ -126,6 +130,11 @@ function createSmoothStreamingParser(
   const MIN_REPRESENTATION_BITRATE = parserOptions.minRepresentationBitrate ||
     0;
 
+  const { serverSyncInfos } = parserOptions;
+  const SERVER_TIME_OFFSET = serverSyncInfos ?
```

Comment:
I don't really get why we are using an `UPPER_CASE` syntax in that function. It denotes from the rest of the code.